content/resources/titlebar.js

```diff
@@ -80,7 +80,7 @@ dx.json('/meta.json', function (err, meta) {
     }
   }
 
-  if (meta.selectedVersion) {
+  if (window.location.pathname.indexOf('/docs/') > -1 && window.location.pathname.indexOf('changelog') === -1) {
```

Comment:
Not sure if this is the best way to do this - it works for hexagon as all the versioned content is located under 'docs'